src/model/orca-pool.ts

```diff
@@ -43,5 +43,11 @@ class OrcaPoolImpl implements OrcaPool {
         }
 
         return shiftByDecimal(result.amount, this.poolParams.poolTokenDecimals).toNumber()
-    };
+    }
+
+    public async getLPSupply(): Promise<number> {
+        const context = await this.connection.getTokenSupply(this.poolParams.poolTokenMint)
+        const amt = new u64(context.value.amount)
+        return shiftByDecimal(amt, this.poolParams.poolTokenDecimals).toNumber()
```

Comment:
Nitpick, but looks like this repo could use a linter to automatically add semicolons and stuff

Reply:
good point. let me consolidate our eslint so we can be consistent (I killed the semi-colon because I'm used to swift)

Reply:
Ohhh, haha :rofl: